python/test/unit/language/test_core.py

```diff
@@ -442,7 +442,7 @@ def do_test(x, y, kernel_fn):
         # triton result
         x_tri = x if x_is_scalar else to_triton(x, device=device, dst_type=dtype_x)
         y_tri = y if y_is_scalar else to_triton(y, device=device, dst_type=dtype_y)
-        z_tri = to_triton(np.empty(SIZE, dtype=z_ref.dtype), device=device)
+        z_tri = to_triton(np.empty(SIZE, dtype=str(z_ref.dtype)), device=device)
```

Comment:
To avoid the issue: `TypeError: can't convert np.ndarray of type numpy.ulonglong. The only supported types are: float64, float32, float16, complex64, complex128, int64, int32, int16, int8, uint64, uint32, uint16, uint8, and bool.`  String name for `numpy.ulonglong` type is `uint64`.